bot/iut_config/mynewt.py

```diff
@@ -22,6 +22,12 @@
         ],
     },
 
+    "public_address.conf": {
+        "overlay": {' BLE_PUBLIC_DEV_ADDR': '(uint8_t[6]){0xff, 0xff, 0xff, 0xff, 0xff, 0xff}'},
```

Comment:
Why do we need to use public address for those tests in first place?

Reply:
ok, I deleted this commit - this passes with CASE0072061